src/fileformat/types/import_table/import_table.cpp

```diff
@@ -820,11 +825,19 @@ void ImportTable::computeHashes()
 		//}
 	}
 
-	if (impHashBytes.size())
-	{
-		impHashCrc32 = getCrc32((const uint8_t *)impHashBytes.data(), impHashBytes.size());
-		impHashMd5 = getMd5((const uint8_t *)impHashBytes.data(), impHashBytes.size());
-		impHashSha256 = getSha256((const uint8_t *)impHashBytes.data(), impHashBytes.size());
+	if (impHashBytes.size()) {
+		auto data = reinterpret_cast<const uint8_t*>(impHashBytes.data());
+
+		Tlsh tlsh;
```

Comment:
As I looked into these hashes for this review, I realized this entire `import_table.cpp` implementation is just for PE. It does not work for ELF at all. Once we merge this PR, I will refactor it to some PE-specific module. As such I don't think telfhash is needed here, it just confuses the thing even more -> remove its computation here.

Reply:
This import table is used also by Mach-O.